src/bin/firebase.js

```diff
@@ -1,7 +1,22 @@
 #!/usr/bin/env node
 "use strict";
 
+// Make check for Node 6, which is no longer supported by the CLI.
+var semver = require("semver");
 var pkg = require("../../package.json");
+var nodeVersion = process.version;
+if (!semver.satisfies(nodeVersion, pkg.engines.node)) {
+  console.error(
```

Comment:
Don't need to do this in this PR but this would look nicer with formatted string

Reply:
I'm going to leave this for now. This will continue to work with older versions of node which would be important (rather than throwing an error like "unknown character "\`"